docs/syntax_by_example.md

```diff
@@ -121,64 +117,69 @@ aspects of `evy`'s syntax. For a formal language specification see the
 
 ### Variadic
 
-    func my_print args:any...
-        for arg := range args
-            write arg
-            write " "
+    func list args:any...
+        for arg := range args[:-1]
+            printf "%v, " arg
         end
+        printf "%v" args[-1]
     end
 
+### Function calls
+
+    n := add 1 2        // 3
+    foxprint "🐾"       // 🦊 🐾  
+    list 2 true "blue" // 2, true, blue
```

Comment:
nit: this comment does not line up with the ones above it

Reply:
it lines up with on the output (I actually changed it).... I will change it back.

Reply:
done.